src/cli/cli.cpp

```diff
@@ -541,6 +541,26 @@ template <> otError Interpreter::Process<Cmd("ba")>(Arg aArgs[])
         }
     }
 #endif // OPENTHREAD_CONFIG_BORDER_AGENT_EPHEMERAL_KEY_ENABLE
+    /**
+     * @cli ba counters
+     * @code
+     * border agent counters
+     * ePSKc: activations 2 api_deactivations 1 timeout_deactivations 1 max_attempt_deactivations 0
+     *        disconnect_deactivations 0
+     * ePSKc: invalid_ba_state_errors 0 invalid_args_errors 1 start_secure_session_errors 0
+     * ePSKc: secure_session_successes 0 secure_session_failures 0 commissioner_petitions 0
+     * PSKc: secure_session_successes 0 secure_session_failures 0 commissioner_petitions 0
```

Comment:
nit: this seems somewhat hard to read and parse. How about giving each counter its own line, just like we do with `counters mac`?

Reply:
Updated to one line per counter, and prefixed counters with `epskc` or `pskc` since there are some common counters for them.